Demo/Demo/FeatureViewControllers/PayPalDemoViewController.swift

```diff
@@ -1,10 +1,76 @@
 import UIKit
+import PayPal
+import PaymentsCore
 
 class PayPalDemoViewController: FeatureBaseViewController {
 
+    // MARK: - SDK Setup
+
+    let config = CoreConfig(clientID: DemoSettings.clientID, environment: .sandbox)
+
+    // MARK: - UI Components
+
+    lazy var payPalButton: CustomButton = {
+        let payPalButton = CustomButton(title: "PayPal")
+        payPalButton.addTarget(self, action: #selector(didTapPayPalButton), for: .touchUpInside)
+        payPalButton.layer.cornerRadius = 8
+        payPalButton.backgroundColor = .systemBlue
+        payPalButton.tintColor = .white
+        payPalButton.isEnabled = false
+        return payPalButton
+    }()
+
+    // MARK: - View Lifecycle & UI Setup
+
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        view.backgroundColor = UIColor.purple
+        view.addSubview(payPalButton)
+        view.backgroundColor = .systemBackground
+
+        setupConstraints()
+    }
+
+    private func setupConstraints() {
+        NSLayoutConstraint.activate([
+            payPalButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: Constants.layoutSpacing),
+            payPalButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -Constants.layoutSpacing),
+            payPalButton.centerYAnchor.constraint(equalTo: view.centerYAnchor),
+            payPalButton.heightAnchor.constraint(equalToConstant: Constants.textFieldHeight)
+        ])
+    }
+
+    // MARK: - FeatureBaseViewController Override
+
+    /// Enable the PayPal Button once we have created an order
+    override func createOrder(completion: @escaping (String?) -> Void = { _ in }) {
+        super.createOrder { orderID in
+            if orderID != nil {
+                self.payPalButton.isEnabled = true
+            } else {
+                print("There was an error")
+            }
+        }
+    }
+
+    // MARK: - PayPal Module Integration
+
+    @objc func didTapPayPalButton() {
+        checkoutWithPayPal(orderID: orderID ?? "")
+    }
+
+    func checkoutWithPayPal(orderID: String) {
+        let payPalClient = PayPalClient(config: config, returnURL: "")
+        payPalClient.start(orderID: orderID, presentingViewController: self) { [weak self] state in
+            guard let self = self else { return }
```

Comment:
You can avoid this guard if the below implementation will be always like that, what do you think?

Reply:
Yeah we could just do `self?...` self when it's used below, but I think adding a guard here is slightly more safe. At the end of the day though I think this may just come down to preference. 🤷‍♂️

Reply:
Yeh - especially since this is just for the Demo app, it doesn't matter a ton. I agree the guard is safer.